third_party/icu/BUILD.gn

```diff
@@ -375,22 +378,34 @@ if (is_android && enable_java_templates) {
   }
 }
 
-if (is_android) {
-  # Use android_small for now to keep the size till we decide to switch to the new one.
-  data_dir = "android_small"
-} else if (is_ios) {
-  data_dir = "ios"
-} else if (is_chromeos) {
-  data_dir = "chromeos"
+if (use_cobalt_customizations) {
+  data_dir = "cobalt"
 } else {
-  data_dir = "common"
+  if (is_android) {
```

Comment:
nit: don't indent the upstream code so there's no diff for those lines

Reply:
This upstream code will be equally indented whether it's in the `if` or the `else` block.

Are you suggesting we leave the upstream code outside of any conditional and then below it, reassign `data_dir` if `use_cobalt_customizations`? I think it's probably more clear without doing a reassignment.

Reply:
No I'm just suggesting to change the whitespace so it doesn't have a diff from upstream but keep everything functionally the same, like in https://github.com/youtube/cobalt/blob/main/third_party/boringssl/BUILD.gn#L130. Just makes it easier to work on minimizing the diff from upstream as we don't have whitespace differences we have to look at

Reply:
Oh, got it. Done.